### ISSUE TITLE
Empty folders & Jellyfin

### ISSUE THREAD

**Issue by bonswouar** [Schaka/janitorr#78]
(that should be my last issue! 😄 )

So I've just noticed that if a library folder is empty (ie `leaving-soon/movies/media`), movies that were previously in this folder will not be removed from Jellyfin.

Jellyfin logs say:
`MediaBrowser.Controller.Entities.BaseItem: Library folder /data/media/leaving-soon/movies/media is inaccessible or empty, skipping`

And after searching for it, it seems **it's a feature not a bug**: https://github.com/jellyfin/jellyfin/issues/11913
Basically to prevent medias from an unavailable mount point to disappear

Apparently one workaround would be:
> If you need it to get cleared up, placing any non-media file there should do the trick.

Would it be possible to implement this in Janitorr?
Basically always create all folders structures, and add some non-media file if they're empty _(or even always add a non-media file I guess, if that makes it easier)_

Thanks!

**Comment by Schaka** [comment]
Sure, that makes sense. I'll probably go with the "always" option. Doesn't hurt to have some empty file in those folders.

**Comment by Schaka** [comment]
In https://github.com/Schaka/janitorr/commit/8f21c3dcf01b5adb9d42391b229ed18119ad9374
